docs/customize/custom_fields/records.md

```diff
@@ -395,9 +395,9 @@ This section lists the field types and UI widgets that are available in InvenioR
 
 ### Field types
 
-- `KeywordCF` for text that need to be searchable **only** as "exact match" (e.g. search filters/facets).
+- `KeywordCF` for text that needs to be searchable **only** as "exact match" (e.g. search filters/facets).
 - `TextCF` for normal text, if you need to search as "exact match" (e.g. search filters/facets) pass `use_as_filter=True` as parameter.
-- `VocabularyCF` for controlled vocabularies. Note that it supports only generic vocabularies (e.g. _names_, _awards_, etc. cannot be linked to a custom field).
+- `VocabularyCF` for controlled vocabularies. Note that it supports only generic vocabularies (e.g. _names_, _awards_, etc.). It cannot be linked to a custom field.
```

Comment:
I'm not sure this change is correct. I read this as indicating that more complex vocabularies like names and awards can't be linked using this field type. Since I haven't tested this, I'm not sure which is correct. In any event is should be rewritten to be clearer.

Reply:
@tmorrell is correct, the line meant that the vocabularyCF cannot be used for non-generic vocabs (e.g. names, awards, etc.). However, it does read badly, should be rewritten.